src/modules/http-exchange.ts

```diff
@@ -46,7 +53,7 @@ export const httpExchange = (): IExchange => operation => {
 
     fetch(url, {
       body,
-      headers: { 'Content-Type': 'application/json' },
+      headers: defaultHeaders,
```

Comment:
So sorry for being unclear in the issue and the comment before :(

I'm not sure whether I see the benefit of: `fetchOptions: ((defaults: RequestInit) => RequestInit);` where the input are defaults. I believe we should just keep spreading it in as below (`...fetchOptions`) and below it add:

```js
headers: {
  'Content-Type': 'application/json',
  ...fetchOptions.headers
}
```

I see the intent of deduping it there, but since case-sensitivity doesn't matter, I'd expect the eventual XHR/fetch to dedupe it.

`(defaults: RequestInit) => RequestInit` is nice in theory but I'm not sure it adds a lot of value. Any thoughts?

Reply:
I agree.  Seems much cleaner as well and does not put the responsibility on the library user.